Gruntfile.js

```diff
@@ -45,54 +53,120 @@ module.exports = function( grunt ) {
       ],
       tests: {
         options: {
-          jquery: true,
-          globals: {
-            Modernizr: true,
-            define: true,
-            TEST: true,
-            QUnit: true
-          }
-        },
-        files: {
-          src: ['test/js/*.js']
-        }
-      },
-      lib: {
-        options: {
-          node: true
+          jshintrc: true
         },
         files: {
-          src: ['lib/*.js']
+          src: [
+            '<%= env.nodeTests%>',
+            '<%= env.browserTests %>'
+          ]
         }
       }
     },
     clean: {
-      dist: ['dist']
+      dist: [
+        'dist',
+        'test/coverage'
+      ]
     },
     connect: {
       server: {
         options: {
+          middleware: function(connect, options) {
+            return [
+              require('connect-jade-static')({
+                baseDir: options.base,
+                baseUrl: '/',
+                jade: {
+                  unitTests: browserTests,
+                  integrationTests: grunt.file.expand(['test/browser/integration/*.js'])
+                }
+              }),
+              function(req, rep, next) {
```

Comment:
Nit picky, but my brain had to pause for a second to understand what `rep` was I think `res` is more obvious.